api/src/main/java/org/pmiops/workbench/monitoring/views/GaugeMetric.java

```diff
@@ -19,19 +21,25 @@
   BILLING_BUFFER_CREATING_PROJECT_COUNT(
       "billing_project_buffer_creating_project_count",
       "Current number of billing projects with creating status."),
+  DEBUG_CONSTANT_VALUE("debug_constant_value", "Always 101"),
   DEBUG_MILLISECONDS_SINCE_EPOCH("debug_epoch_millis", "Number of milliseconds since epoch"),
   DEBUG_RANDOM_DOUBLE(
       "debug_random_double",
       "Double value for debugging",
       "MHz",
       MeasureDouble.class,
       Aggregation.Sum.create()),
-  NOTEBOOK_SAVE("notebook_save", "Save (or create) a notebook"),
-  NOTEBOOK_CLONE("notebook_clone", "Clone (duplicate) a notebook"),
-  NOTEBOOK_DELETE("notebook_delete", "Delete a notebook"),
   WORKSPACE_TOTAL_COUNT("workspace_total_count", "Count of all workspaces (including inactive)"),
+  WORKSPACE_COUNT_BY_ACTIVE_STATUS(
+      "workspace_count_by_active_status", "Count of workspaces by active status"),
+  WORKSPACE_COUNT_BY_DATA_ACCESS_LEVEL(
+      "workspace_count_by_data_access_level", "Count of workspaces for each data access level"),
   DATASET_COUNT("dataset_count", "Count of all datasets in existence"),
-  COHORT_COUNT("cohort_count", "Count of all cohorts in existence");
+  DATASET_COUNT_BY_INVALID("dataset_count_by_invalid", "Count of all datasets by invalid status"),
```

Comment:
[nit] Can you keep this list of enums alphabetized? Otherwise it's going to get pretty intractable.

Reply:
I've been thinking about this a bunch. Java doesn't let me split an enum into multiple files, and it also doesn't allow enums to extend abstract classes or other enums (or any classes). Any of those would give me an easy way to break things down.

As it is, having more than one GaugeMetric enum would involve some duplication of code between the files. For example, if the enum is going to have constructor calls, the constructor needs to be defined in that enum class.

But more to the point, if I have aspirations to make this Swagger/Spring adapter a separate library, I need a way to pull out our aou-specific metrics into an input file, like JSON or XML or the other one I can't think of that ain't some other thing. I'll be bummed to lose the strongly-typed metric identifiers, though.

Or a .properties file or something. Basically some way to make the metric definitions part of the configuration instead of the code and safely use them at runtime.

Reply:
Alphabetizing for now.